src/main/java/edu/cornell/kfs/fp/batch/service/impl/TravelMealCardFileFeedServiceImpl.java

```diff
@@ -0,0 +1,266 @@
+        
+        Date loadDate = (tmCardFileLine.getLoadDate() == null || tmCardFileLine.getLoadDate().isBlank()) 
+                ? dateTimeService.getCurrentSqlDateMidnight()
+                : convertToSqlDate(cardServicesUtilityService.changeFormatFromYYYYMMDDToSlashedMMDDYYYY(tmCardFileLine.getLoadDate()));
+        
+        TravelMealCardVerificationData verifyData = new TravelMealCardVerificationData();
+        verifyData.setCardHolderAccountNumber(tmCardFileLine.getCardHolderAccountNumber());
+        verifyData.setEmplid(tmCardFileLine.getEmplid());
+        verifyData.setCardHolderLine1Address(tmCardFileLine.getCardHolderLine1Address());
+        verifyData.setCardHolderLine2Address(tmCardFileLine.getCardHolderLine2Address());
+        verifyData.setCardHolderCityName(tmCardFileLine.getCardHolderCityName());
+        verifyData.setCardHolderStateCode(tmCardFileLine.getCardHolderStateCode());
+        verifyData.setCardHolderPostalCode(tmCardFileLine.getCardHolderPostalCode());
+        verifyData.setCardHolderWorkPhoneNumber(tmCardFileLine.getCardHolderWorkPhoneNumber());
+        verifyData.setCardHolderPersonalPhoneNumber(tmCardFileLine.getCardHolderPersonalPhoneNumber());
+        verifyData.setCreditLine(creditLine);
+        verifyData.setExpireDate(tmCardFileLine.getExpireDate());
+        verifyData.setFileCreateDate(fileCreateDate);
+        verifyData.setLoadDate(loadDate);
+        
+        TravelMealCardCertificationData certifyData = new TravelMealCardCertificationData();
+        certifyData.setCardHolderAccountNumber(tmCardFileLine.getCardHolderAccountNumber());
+        certifyData.setEmplid(tmCardFileLine.getEmplid());
+        certifyData.setCardHolderName(tmCardFileLine.getCardHolderName());
+        certifyData.setCardStatus(tmCardFileLine.getCardStatus());
+        certifyData.setCardType(tmCardFileLine.getCardType());
+        certifyData.setNetid(tmCardFileLine.getNetid());
+        certifyData.setLastTransactionDate(convertToSqlDate(cardServicesUtilityService.changeFormatFromYYYYMMDDToSlashedMMDDYYYY(tmCardFileLine.getLastTransactionDate())));
+        certifyData.setDefaultAccountNumber(tmCardFileLine.getDefaultAccountNumber());
+        certifyData.setCreditLine(creditLine);
+        certifyData.setOpenDate(convertToSqlDate(cardServicesUtilityService.changeFormatFromYYYYMMDDToSlashedMMDDYYYY(tmCardFileLine.getOpenDate())));
+        certifyData.setCycleStartDate(convertToSqlDate(cardServicesUtilityService.changeFormatFromYYYYMMDDToSlashedMMDDYYYY(tmCardFileLine.getCycleStartDate())));
+        certifyData.setCycleSpendToDate(cardServicesUtilityService.generateKualiDecimal(tmCardFileLine.getCycleSpendToDate()));
+        certifyData.setActivationStatus(tmCardFileLine.getActivationStatus());
+        certifyData.setFileCreateDate(fileCreateDate);
+        certifyData.setLoadDate(loadDate);

```

Comment:
I think an idea for making this method easier to read could be to add a helper method that calls changeFormatFromYYYYMMDDToSlashedMMDDYYYY and convertToSqlDate